documentation/src/developer_documentation/mainnet_compatibility.md

```diff
@@ -55,8 +55,17 @@ snapshots only contain stateroot data for the previous 2000 epochs. So, if you
 have a statediff at epoch X, download a snapshot for epoch X+100 and tell Forest
 to re-validate the snapshot from epoch X.
 
-For more detailed instructions, follow
-[this document](https://www.notion.so/chainsafe/Interop-debugging-6adabf9222d7449bbfeaacb1ec997cf8)
+Steps to print a state-diff:
+
+1. Note the epoch of the state-root mismatch. State-roots can only be checked
+   for the parents of a tipset so the failing epoch may be 1 higher than you
+   think.
+2. Download a recent snapshot dated _before_ the failing epoch.
+3. Import the snapshot into Lotus and sync to HEAD.
+4. Export a new snapshot 100 epochs _after_ the failing epoch.
+5. Convert the `.car.zst` file to `.forest.car.zst`.
```

Comment:
For completeness, let's write _how_ to perform this conversion.

Reply:
Done.